api/resume_parser/parser.py

```diff
@@ -0,0 +1,318 @@
+from datetime import date
+
+import PyPDF2
+import fitz
+import re
+import json
+
+
+class ResumeDataExtractor:
+    pdffile = ''
+    text = ''
+    formatted_text = []
+    imp_headings_text = {}
+
+    def __init__(self, file):
+        self.pdffile = file
+        doc = fitz.open(self.pdffile)
```

Comment:
Can you explain what it's doing because it's showing me some errors
![image](https://user-images.githubusercontent.com/47397638/140655553-7d5016fb-f3f4-4aa6-9620-bd6cb67a1b04.png)